### PR TITLE
k8s: add command to list regions/sizes, fix command help descriptions

### PR DESCRIPTION
**Pull request by aybabtme**

### CODE REVIEW

File: commands/displayers/kubernetes.go [hunk 1]
```diff
@@ -204,3 +204,81 @@ func (versions *KubernetesVersions) KV() []map[string]interface{} {
...
+	return map[string]string{
+		"Name": "Name",
+		"Slug": "Slug",
+	}
+}
+
+func (regions *KubernetesRegions) KV() []map[string]interface{} {
+	out := make([]map[string]interface{}, 0, len(regions.KubernetesRegions))
+
+	for _, version := range regions.KubernetesRegions {
```
Comment (nanzhong) [inline commands/displayers/kubernetes.go]: `s/version/region/`

File: commands/displayers/kubernetes.go [hunk 2]
```diff
@@ -204,3 +204,81 @@ func (versions *KubernetesVersions) KV() []map[string]interface{} {
...
+	return map[string]string{
+		"Name": "Name",
+		"Slug": "Slug",
+	}
+}
+
+func (nodeSizes *KubernetesNodeSizes) KV() []map[string]interface{} {
+	out := make([]map[string]interface{}, 0, len(nodeSizes.KubernetesNodeSizes))
+
+	for _, version := range nodeSizes.KubernetesNodeSizes {
```
Comment (nanzhong) [inline commands/displayers/kubernetes.go]: `s/version/size/`

File: commands/kubernetes.go [hunk 3]
```diff
@@ -106,11 +106,11 @@ func kubernetesCluster() *Command {
 	CmdBuilder(cmd, RunKubernetesClusterList, "list", "get a list of your clusters", Writer, aliasOpt("ls"))
 
 	cmdKubeClusterCreate := CmdBuilder(cmd, RunKubernetesClusterCreate(defaultNodeSize, defaultNodeCount), "create <name>", "create a cluster", Writer, aliasOpt("c"))
-	AddStringFlag(cmdKubeClusterCreate, doctl.ArgRegionSlug, "", defaultRegion, "cluster region location, example value: nyc1", requiredOpt())
-	AddStringFlag(cmdKubeClusterCreate, doctl.ArgClusterVersionSlug, "", "", "cluster version")
+	AddStringFlag(cmdKubeClusterCreate, doctl.ArgRegionSlug, "", defaultRegion, "cluster region location. possible values: see `doctl k8s options regions`.", requiredOpt())
+	AddStringFlag(cmdKubeClusterCreate, doctl.ArgClusterVersionSlug, "", "", "cluster version. possible values: see `doctl k8s options versions`")
```
Comment (nanzhong) [inline commands/kubernetes.go]: comma instead of period

File: commands/kubernetes.go [hunk 4]
```diff
@@ -106,11 +106,11 @@ func kubernetesCluster() *Command {
 	CmdBuilder(cmd, RunKubernetesClusterList, "list", "get a list of your clusters", Writer, aliasOpt("ls"))
 
 	cmdKubeClusterCreate := CmdBuilder(cmd, RunKubernetesClusterCreate(defaultNodeSize, defaultNodeCount), "create <name>", "create a cluster", Writer, aliasOpt("c"))
-	AddStringFlag(cmdKubeClusterCreate, doctl.ArgRegionSlug, "", defaultRegion, "cluster region location, example value: nyc1", requiredOpt())
-	AddStringFlag(cmdKubeClusterCreate, doctl.ArgClusterVersionSlug, "", "", "cluster version")
+	AddStringFlag(cmdKubeClusterCreate, doctl.ArgRegionSlug, "", defaultRegion, "cluster region location. possible values: see `doctl k8s options regions`.", requiredOpt())
```
Comment (nanzhong) [inline commands/kubernetes.go]: comma instead of period.

File: commands/kubernetes.go [hunk 5]
```diff
@@ -106,11 +106,11 @@ func kubernetesCluster() *Command {
...
+	AddStringFlag(cmdKubeClusterCreate, doctl.ArgClusterVersionSlug, "", "", "cluster version. possible values: see `doctl k8s options versions`")
 	AddStringSliceFlag(cmdKubeClusterCreate, doctl.ArgTagNames, "", nil, "cluster tags")
-	AddStringFlag(cmdKubeClusterCreate, doctl.ArgSizeSlug, "", defaultNodeSize, "size of the nodes in the default node pool (incompatible with --"+doctl.ArgClusterNodePool+")")
-	AddStringFlag(cmdKubeClusterCreate, doctl.ArgNodePoolCount, "", strconv.Itoa(defaultNodeCount), "size of the nodes in the default node pool (incompatible with --"+doctl.ArgClusterNodePool+")")
+	AddStringFlag(cmdKubeClusterCreate, doctl.ArgSizeSlug, "", defaultNodeSize, "size of the nodes in the default node pool (incompatible with --"+doctl.ArgClusterNodePool+"). possible values: see `doctl k8s options sizes`.")
```
Comment (nanzhong) [inline commands/kubernetes.go]: comma instead of period.

File: commands/kubernetes.go [hunk 6]
```diff
@@ -106,11 +106,11 @@ func kubernetesCluster() *Command {
...
+	AddStringFlag(cmdKubeClusterCreate, doctl.ArgNodePoolCount, "", strconv.Itoa(defaultNodeCount), "number of nodes nodes in the default node pool (incompatible with --"+doctl.ArgClusterNodePool+")")
```
Comment (nanzhong) [inline commands/kubernetes.go]: nodes is repeated, but shouldn't be.

File: commands/kubernetes.go (hunk 4 above)
Comment (joonas) [reply commands/kubernetes.go]: should it be `cluster region` or `cluster location`, not `cluster region location`?

**Comment by aybabtme** [comment]
@nanzhong PTAL

**Pull request merged**